exercises/book-store/description.md

```diff
@@ -0,0 +1,49 @@
+To try and encourage more sales of the 5 different books they sell in a popular series, a bookshop has decided to offer 
+discounts of multiple-book purchases. 
+
+One copy of any of the five books costs 8 EUR. 
+
+If, however, you buy two different books, you get a 5% discount on those two books.
+
+If you buy 3 different books, you get a 10% discount. 
+
+If you buy 4 different books, you get a 20% discount.
+
+If you go the whole hog, and buy all 5, you get a huge 25% discount. 
+
+Note that if you buy, say, four books, of which 3 are different titles, you get a 10% discount on the 3 that 
+form part of a set, but the fourth book still costs 8 EUR. 
+
+Your mission is to write a piece of code to calculate the price of any conceivable shopping basket (containing only books from the same series), giving as big a discount as possible.
+
+For example, how much does this basket of books cost?
+
+2 copies of the first book
+2 copies of the second book
+2 copies of the third book
+1 copy of the fourth book
+1 copy of the fifth book
+
+One way of group these 8 books is:
+ 1 group of 5 --> 25% discount (1st,2nd,3rd,4th,5th)
```

Comment:
These also get mashed together. Not sure of the best way to deal with it - use backticks?